doc/faq.mkd

```diff
@@ -50,3 +50,16 @@ on your remote repositories:
 [bitbucket-default-branch]: https://answers.atlassian.com/questions/280944/how-to-change-main-branch-in-bitbucket "How to change MAIN branch in BitBucket?"
 [gitolite-v2-default-branch]: http://stackoverflow.com/questions/7091599/git-default-remote-branch-with-gitolite "git default remote branch with gitolite"
 [gitolite-v3-default-branch]: http://stackoverflow.com/questions/13949093/git-change-default-branch-gitolite "git change default branch (gitolite)"
+
+#### I get the error: [R10K::Action::Deploy::Environment - ERROR] Couldn't update git cache for ...
```

Comment:
If this error comes up, then it means something bad happened while trying to fetch the backing Git repo - from access denied, to network issues, to the Git binary segfaulting. Is this trying to document a specific issue?